src/common/graphQL/graphqlSchema/post.graphqlSchema.ts

```diff
@@ -55,14 +53,26 @@ ${PostData_Fragment.postResult}
 
 
 export const getPostByIDSchema = gql`
-query($postID: String!){
+query($postID: ID!){
 	searchPostByID(postID:$postID) {
         ...PostRequestData
   }
 }
 ${PostData_Fragment.postResult}
 `;
 
+export const getNextPostIDSchema = gql`
+query($postID: String!) {
```

Comment:
is this postID still string? plz let me know if it is

Reply:
i don't use that schema i will remove